components/dashboard/src/components/Modal.tsx

```diff
@@ -50,9 +50,9 @@ export default function Modal(props: {
     return (
         <div className="fixed top-0 left-0 bg-black bg-opacity-70 z-50 w-screen h-screen" onClick={props.onClose}>
             <div className="w-screen h-screen align-middle" style={{display: 'table-cell'}}>
-                <div className={"relative bg-white border rounded-xl p-6 max-w-lg mx-auto text-left text-gray-600 " + (props.className || '')} onClick={e => e.stopPropagation()}>
+                <div className={"relative bg-white dark:bg-gray-900 border border-gray-200 dark:border-gray-700 rounded-xl p-6 max-w-lg mx-auto text-left text-gray-600 " + (props.className || '')} onClick={e => e.stopPropagation()}>
```

Comment:
**question**: What do you think of using a darker border here and on all borders, like `dark:border-gray-800`? No strong feelings only posting this as an option.

| BEFORE | AFTER |
|-|-|
| ![image](https://user-images.githubusercontent.com/120486/114918289-97102000-9e2f-11eb-9e46-824386f9b863.png) | ![image](https://user-images.githubusercontent.com/120486/114918289-97102000-9e2f-11eb-9e46-824386f9b863.png) |

Reply:
Agree, darker/after looks better. 🖤

Reply:
I'd only hold this back for accessibility reasons but as long as we keep text elements accessible this should be fine. We'll go through any accessibility issues in another Issue/PR. 🧀